bin/fusor-upgrade-qci

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+FUSOR_LIBEXEC_DIR="/usr/libexec/fusor-installer"
+ORG="Default Organization"
+
+echo -e "############################################################\n"
+echo -e " This script will Upgrade your QCI to version 1.1\n"
+echo -e " The upgrade process may take a long time."
+echo -e " Consider using a utility such as 'screen'.\n"
+echo -e " Review the following before proceeding: "
+echo -e "  - Updated system (e.g. 'yum update')"
+echo -e "  - No running tasks in Satellite"
+echo -e "  - Back up Databases\n"
+echo -e "############################################################\n"
+
+read -p "Do you want to continue? [y/N] " -n 1 -r
+if [[ $REPLY =~ ^[Yy]$ ]]; then
+
```

Comment:
I would've done the check for N and exited, then you wouldn't have to indent all of the YES scenario code.
```
if NOT == to N; then
   echo -e "\n\nUser ABORT! Not running the QCI upgrade script.. and exiting..\n"
fi

# User selected YES so we're upgrading
PUT REST OF THE CODE HERE
```